cypress/integration/editPage.spec.js

```diff
@@ -196,7 +199,7 @@ describe("Edit unlinked page", () => {
     cy.get("#modal-delete").click()
 
     // Assert: page no longer exists
-    cy.visit(`/sites/${TEST_REPO_NAME}/pages/${TEST_UNLINKED_PAGE_FILE_NAME}`)
+    cy.visit(`/sites/${TEST_REPO_NAME}/editPage/${TEST_PAGE_TITLE_ENCODED}`)
     cy.contains("The page you are looking for does not exist anymore.")
```

Comment:
Could we look into increasing the timeout for this test? It's consistently failing for me

Reply:
good idea! fixed here 6b62b78